trace/id.go

```diff
@@ -17,6 +17,7 @@ type Generator interface {
 
 var traceIDGen Generator
 
+// nolint: gosec
```

Comment:
Why is this here? could you comment on what was the issue with the linter, and why it can be ignored?

Maybe you can put this directly over the line that caused the problem? I feel that gosec is too important to ignore in the whole function.

Reply:
gosec complains about math.Rand instead of crypto.Rand, but here it's ok, because math.Rand is faster and we are not using Rand for security purpose.. I will add this explanation as a comment also.